dev-tools/ecs-migration.yml

```diff
@@ -47,6 +47,11 @@
   alias6: true
   alias: true
 
+- from: beat.timezone
+  to: event.timezone
+  alias6: false
```

Comment:
Nit: This will conflict with #9283. It's actually `alias6:true` as well ;-)

Reply:
correct, fixed.